node/cmd/guardiand/node.go

```diff
@@ -392,10 +392,10 @@ func runNode(cmd *cobra.Command, args []string) {
 		if *ethRopstenContract != "" {
 			logger.Fatal("Please do not specify --ethRopstenContract in non-testnet mode")
 		}
-		if *fantomRPC == "" {
+		if *fantomRPC != "" && !*unsafeDevMode {
 			logger.Fatal("Please do not specify --fantomRPC in non-testnet mode")
 		}
-		if *fantomContract == "" {
+		if *fantomContract != "" && !*unsafeDevMode {
```

Comment:
unsafe devnet mode AND testnet are different things I thought? I'd assumed that testnet meant literal whatever network testnet and devnet meant local tilt network. We don't run a fantom container for devnet. Is this the intended solution?

Reply:
@leoluk ping